notebooks_tsqr/NightLog.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "0",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if gaps:
</pre><p>sketches of our ideas for this to bring up wednesday</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
https://rubinobs.atlassian.net/browse/DM-46482